### PR TITLE
add: avatar css as introduction.css

### PR DESCRIPTION
**Pull request by shilpe26**
Made avatar component with features like:-
🔹 Avatar with the profile picture
🔹 Avatar with different sizes
🔹 With code using Carbon 

avatar

### CODE REVIEW

File: css/component_styles/introduction.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+.aside-nav{
+    padding: 2.5rem;
+width: 20rem;
+font-size: 2rem;
+height: 40rem;
+overflow-y: auto;
+}
+/* ::-webkit-scrollbar{
+    display: none;
+} */
```
Comment (Maianki) [inline css/component_styles/introduction.css]: Remove the unused code, don't comment it.

File: css/component_styles/introduction.css [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+}
+.avatar-text{
+    font-size: 1.8rem;
+    padding: 1rem;
+margin-top: -1rem;
+width: 85rem;
+}
+.avatar-text span{
+    color: var(--primary-color);
+}
```
Comment (Maianki) [inline css/component_styles/introduction.css]: Be consistent with the indentation. Everything else looks good.

**Review by ArjunGTX — CHANGES_REQUESTED** [review]
Use extentions like prettier to format the code and keep it consistent. Rest looks good.

File: css/component_styles/introduction.css (hunk 2 above)
Comment (shilpe26) [reply css/component_styles/introduction.css]: 👉🏻Remove the unwanted comments.

🤝🏻Thanks you Maianki for reviewing and suggesting the goodwill.

**Comment by shilpe26** [comment]
> Use extentions like prettier to format the code and keep it consistent. Rest looks good.

**Comment by shilpe26** [comment]
Beautify the file so that code looks good.

**Pull request merged**